code/modules/mob/mob_movement.dm

```diff
@@ -394,4 +396,14 @@
 	return
 
 /mob/proc/update_gravity()
-	return
\ No newline at end of file
+	return
+
+/mob/Move(atom/newloc, direct) // Buckling 
+	. = ..()
+	if(buckled_mob)
+		if(!buckled_mob.Move(loc, direct))
+			loc = buckled_mob.loc
+			last_move = buckled_mob.last_move
+			inertia_dir = last_move
+			buckled_mob.inertia_dir = last_move
+			. = 0
```

Comment:
I think this is copypasta from bed and mulebot Move, is there a way to move all three at atom/movable/Move() ?